### PR TITLE
Tweak success stories

### PR DESCRIPTION
**Pull request by leosoaivan**
* Remove completed course text from success stories card
* Bold student_name in success stories card
* Include job title beside name, not bold
* Change margin-bottom for name section to 10px
* Stack and center avatar and text on success stories card

### CODE REVIEW

**Comment by leosoaivan** [comment]
:tada: first non-grammar/content PR :tada:

**Comment by KevinMulhern** [comment]
👍 LGTM
Thank you @leosoaivan 🎉

**Comment by leosoaivan** [comment]
Removed some information on the database.yml :p

File: config/database.yml [hunk 1]
```diff
@@ -5,6 +5,8 @@ development:
   adapter: postgresql
   database: theodinproject
   host: localhost
+  username:
```
Comment (KevinMulhern) [inline config/database.yml]: Looks like theres the database.yml is still in the PR @leosoaivan 😛 

You can get it out of the pr by checking out the master version of the file in your branch locally:
`git checkout master config/database.yml`

and then add and make a new commit

Comment (leosoaivan) [inline config/database.yml]: Thanks, yeah. I don't recall ever staging it for a commit, so I was surprised to see this myself? Will get on it ASAP.

**Comment by KevinMulhern** [comment]
Perfect thanks @leosoaivan 😄

**Pull request merged**